recipes/phylogenize/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.91" %}
+{% set sha256 = "3d428440f7a8aed3be18bbe094d3afe94f2d5357daa715d13e963c665384f2ab" %}
+{% set name = "phylogenize" %}
+
+package:
+  name: phylogenize
+  version: "{{ version }}"
+
+source: 
+  url: https://github.com/pbradleylab/phylogenize/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
`{{ pin_subpackage("phylogenize", max_pin="x") }}`

Reply:
I think the name variable from line 3 will sub into the pinning right?

Reply:
nope, see https://bioconda.github.io/contributor/linting.html#missing-run-exports

Reply:
Yes, I believe it says in there that you can use the name variable.

"Case 1: If the software follows semantic versioning (or it has at least a normal version string (like 1.2.3) and the actual strategy of the devs is unknown), add run_exports to the recipe like this:

build:
  run_exports:
    - {{ pin_subpackage('myrecipe', max_pin="x") }}
with myrecipe being the name of the recipe (you can also use the name variable). This will by default pin the package to >=1.2.0,<2.0.0 where 1.2.0 is the version of the package at build time of the one depending on it and <2.0.0 constrains it to be less than the next major (i.e. potentially not backward compatible) version."

Reply:
Ah, my bad, sorry, I was not aware of that.